WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -278,6 +278,20 @@ def update_events
     end
   end
 
+  def update_wcif
+    @competition = competition_from_params
+    wcif = params.permit!.to_h
```

Comment:
Hell yes, that name makes everything much better!!

Still a little bothered by the lack of sanitation of the possible Gaga related input. Maybe I shouldn't be?